print/print-apps/oereb/topicpage.jrxml

```diff
@@ -397,7 +396,7 @@
 				<subreportParameter name="Theme_Text">
 					<subreportParameterExpression><![CDATA[$F{Theme_Text}]]></subreportParameterExpression>
 				</subreportParameter>
-				<dataSourceExpression><![CDATA[$F{ArticleDataSource}]]></dataSourceExpression>
+				<dataSourceExpression><![CDATA[$F{ReferenceDataSource}]]></dataSourceExpression>
```

Comment:
Why does that change from ArticleDataSource to ReferenceDataSource? Did you have an old version?

Reply:
Because we need to switch it. I think we have a misleading naming of variables here. As far as I can see this, it solves the problem which was adressed in #407 and #398 first point as well. This was not solved in your pull request #433

Reply:
Is this answering your question? Or should I change something here?

Reply:
ok, I get it. thanks 👍